scripts/wait-for-published-packages.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-undef */
+const waitOn = require('wait-on');
+const yargs = require('yargs/yargs');
+const {hideBin} = require('yargs/helpers');
+const {getUiTemplates} = require('./get-ui-templates');
+
+async function waitForPackage(packageName, version) {
+  const opts = {
+    resources: [`https://www.npmjs.com/package/${packageName}/v/${version}`],
+    delay: 1e3, // initial delay in ms
+    interval: 5e3, // poll interval in ms
+    timeout: 5 * 60 * 1e3, // timeout in ms
+    window: 1e3, // stabilization time in ms
+    log: true,
+  };
```

Comment:
Waiting for the packages to be ready. I don't know if a timeout of 5 minutes is enough.